wiki/markdown/configuration.md

```diff
@@ -141,6 +141,36 @@ used. **er.url** takes a full URL to such a repository.
 
 * er.url
 
+## RabbitMQ
+
+You can configure the RabbitMQ settings using the rabbitmq.* properties.
+Most of the properties should be familiar but a few may need some further explanation.
+The rabbitmq.domainId, rabbitmq.componentName, rabbitmq.consumerName and rabbitmq.queue.durable
+are used to build the queue name on which Eiffel Intelligence listens for outside messages.
+Multiple routing keys can be defined for this queue with the rabbitmq.binding.key property by writing
```

Comment:
It's a bit confusing when both bindingkeys and routing keys are used interchangeably in the text. We call the property bindingkey so should we refer to it like that also in the documentation?

Reply:
I don't know. They are routing keys but they are used to bind with since Eiffel Intelligence does not send any messages itself. The language is the same in master. Do you want me to change this?

Reply:
Ok, then keep it as is